src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const logoEl = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  const messageEl = document.createElement('div');
+
+  messageEl.classList.add('message');
+  messageEl.textContent = 'Promise was resolved!';
+  logoEl.onclick = () => resolve(messageEl);
+});
+
+promise1.then((messageEl) => document.body.append(messageEl));
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    const errorEl = document.createElement('div');
+
+    errorEl.classList.add('message', 'error-message');
+    errorEl.textContent = 'Promise was rejected!';
+    reject(errorEl);
+  }, 3000);
+});
+
+promise2.then((messageEl) => document.body.append(messageEl));
```

Comment:
The success handler for `promise2` is not necessary because `promise2` is designed to be rejected after 3 seconds. You should remove the `.then()` method for `promise2`.